osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -45,9 +46,11 @@ public abstract class SongSelect : OsuScreen, IKeyBindingHandler<GlobalAction>
     {
         public static readonly float WEDGE_HEIGHT = 245;
 
-        protected const float BACKGROUND_BLUR = 20;
+        protected float BackgroundBlur = 20;
         private const float left_area_padding = 20;
 
+        private Bindable<float> backgroundBlurBindable { get; set; }
```

Comment:
move this to `ScreenWithBeatmapBackground`

Reply:
Shouldn't it be moved to `BackgroundScreenBeatmap`?

Reply:
That's the one, sorry for the confusion.

Reply:
Sorry if im missing something but it seems `userBlurLevel` is tied to `OsuSetting.BlurLevel` which is ingame blur right?

Reply:
Yes, well you're going to have to figure out how to make this work. There's gameplay blur and UI blur now.

The class handles the case where user blur override can be enabled or not. The blur you're wanting to add is the case where "user blur" is "not applied". But now you're adding a new user blur kind.

It will probably involve renaming and refactoring the existing code to ensure it is understandable given the new complexities.

Reply:
Should it be added in `UserDimContainer` where `EnableUserDim` is, or in `BackgroundScreenBeatmap` as an additional boolean that overrides `EnableUserDim`?

Reply:
`UserDimContainer` is specific to gameplay. It should be in `BackgroundScreenBeatmap` as above, in the case that "EnableUserDim" is not set. Although again, you need to rename "UserDim" to "GameplayDim" or something.

Reply:
`backgroundModeBeatmap.BlurAmount` is directly set in `SongSelect`, seems like adding the option to `BackgroundScreenBeatmap` won't actually work because its set to multiple different values (ex `20` in SongSelect` and `15` in `PlayerLoader`)

Reply:
Nevermind, looks like i'm just a bit dumb